akka-cluster/src/test/scala/akka/cluster/ClusterTestKit.scala

```diff
@@ -131,5 +134,76 @@ trait ClusterTestKit extends TestKitBase {
       if (firstSeedNode) newActorSystemAsFirst(newConfig)
       else newActorSystem(newConfig)
     }
+
+    /**
+     * Returns true if the cluster instance for the provided [[ActorSystem]] is [[MemberStatus.Up]].
+     */
+    def isMemberUp(system: ActorSystem): Boolean = Cluster(system).readView.status == MemberStatus.Up
+
+    /**
+     * Returns true if the cluster instance for the provided [[ActorSystem]] has be shutdown.
+     */
+    def isTerminated(system: ActorSystem): Boolean = Cluster(system).readView.isTerminated
+
   }
 }
+
+abstract class RollingUpgradeClusterSpec(config: Config) extends AkkaSpec(config) with ClusterTestKit {
```

Comment:
this is only used in JoinConfigCompatCheckerRollingUpdateSpec so should it be defined there? or do you foresee that we will use this for more tests?

Reply:
I put it there so that it can be reused, because yes I do see it for more tests.

Reply:
It's in two specs not just one currently.